src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -180,30 +185,31 @@
 public class Datasets extends AbstractApiBean {
 
     private static final Logger logger = Logger.getLogger(Datasets.class.getCanonicalName());
-    
-    @Inject DataverseSession session;    
+
+    @Inject
+    DataverseSession session;
 
     @EJB
     DatasetServiceBean datasetService;
 
     @EJB
     DataverseServiceBean dataverseService;
-    
+
```

Comment:
This file appears to have been completely reformatted, which is suboptimal. Usually we try to reformat just the code we touch: https://guides.dataverse.org/en/5.6/developers/coding-style.html#format-code-you-changed-with-netbeans

I still dream of us adopting a tool to format our code for us like gofmt or rustfmt: https://groups.google.com/g/dataverse-dev/c/y2Jpk3szTf8/m/rckKmP6-BgAJ